schemas/2015-08-01/Microsoft.Compute.json

```diff
@@ -22,11 +22,25 @@
           "type": "object",
```

Comment:
Can you support expression for this "properties" too? (L21), currently it is failing ResourceMetaSchema.json validation.